nginx.tmpl

```diff
@@ -1,3 +1,5 @@
+{{ $CurrentContainer := where $ "ID" .Docker.CurrentContainerID | first }}
```

Comment:
@jwilder  Thanks, I'm currently working to run the tests.

@md5 Yes of course, but it's a problem. The best use case stays to use docker-gen and nginx in the same container. Currently, the only test which fails is docker.bats (nginx + dcoker-gen + nginx.tpml). I works on and I'm come back make a checkpoint.